loadtest/control/utils.go

```diff
@@ -57,16 +57,28 @@ func RandomizeUserName(name string) string {
 }
 
 func emulateUserTyping(t string, cb func(term string) UserActionResponse) UserActionResponse {
+	r := rand.New(rand.NewSource(time.Now().Unix()))
```

Comment:
This is a general question to both @agnivade and @isacikgoz:

do we need to create additional sources or should we just rely on the global one?

At some point we'll have to call `rand.Seed()` when starting the program. I was waiting to end the main coding/debug process before doing that and probably should always have a way to disable it through an env variable.

Reply:
Hmm, good question! Actually, that was the reason I didn't use seed initially, because every other `rand.Intn()` function that I came across was not seeded.

Reply:
My understanding is that you use `NewSource` if you want to make some behaviour deterministic given that the global one could have been changed at any point in the program's flow.

I think for our current case we can rely on setting the global source once at the start of the program and rely on that.

I can make a separate ticket to address this.

Reply:
Yes you are right: [rand.go#L303](https://github.com/golang/go/blob/20a838ab94178c55bc4dc23ddc332fce8545a493/src/math/rand/rand.go#L303). If using a single seed will grant us required randomness, we may use that way.

Agreed on addressing with another ticket. 👍

Reply:
Yep, we need to call rand.Seed at a global level. There's no need to initialize new sources every time we call rand.Intn().

Reply:
Very well :+1: 
@isacikgoz if you could remove this line then we can merge and address seeding the global generator in a separate PR. Thanks!